vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidvci/CredentialIssuer.kt

```diff
@@ -159,7 +160,8 @@ class CredentialIssuer(
      *
      * @param user used to create the credential when the wallet app requests the credential
      */
-    // TODO Migrate this to OAuth2AuthorizationServerAdapter?
+    @Suppress("DEPRECATION")
```

Comment:
?

Reply:
nvm. although i would prefer the annotation right before the statement that causes a warning